docs/sbk-internals.md

```
@@ -1753,21 +2349,22 @@ sequenceDiagram
     Note over Bench: future.complete(null) then main exits
 ```
 
-The exact format of the final stdout line (lifted from a real ObjectScale
-run earlier in this session):
+An abbreviated example of the final stdout line is:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)